tests/system/test_query.py

```diff
@@ -1291,36 +1291,38 @@ class SomeKind(ndb.Model):
     assert results[0].foo == 1
 
 
-@pytest.mark.usefixtures("client_context")
-def test_map(dispose_of):
-    class SomeKind(ndb.Model):
-        foo = ndb.StringProperty()
-        ref = ndb.KeyProperty()
-
-    class OtherKind(ndb.Model):
-        foo = ndb.StringProperty()
-
-    foos = ("aa", "bb", "cc", "dd", "ee")
-    others = [OtherKind(foo=foo) for foo in foos]
-    other_keys = ndb.put_multi(others)
-    for key in other_keys:
-        dispose_of(key._key)
-
-    things = [SomeKind(foo=foo, ref=key) for foo, key in zip(foos, other_keys)]
-    keys = ndb.put_multi(things)
-    for key in keys:
-        dispose_of(key._key)
-
-    eventually(SomeKind.query().fetch, _length_equals(5))
-    eventually(OtherKind.query().fetch, _length_equals(5))
-
-    @ndb.tasklet
-    def get_other_foo(thing):
-        other = yield thing.ref.get_async()
-        return other.foo
-
-    query = SomeKind.query().order(SomeKind.foo)
-    assert query.map(get_other_foo) == foos
+# TODO: the return with values inside the tasklet crashes tests on Python 2.7
```

Comment:
You can just convert these to use "raise Return(x)" instead of "return x". Or if we want to test both, we can run different tests for python 2 and python 3. "@pytest.mark.skipif" is handy for that.

Reply:
Turns out it errors out before even running the tests if the return is there, but I'll use the raise Return method.